docs/Guides/Scim/Patch/SCIM-Patch-supported-operations.md

```diff
@@ -0,0 +1,124 @@
+---
+stoplight-id: fdxg63veh82v5
+---
+
+# SCIM Patch supported operations 
+
+
+Patch for email is not supported. Email is primary attribute, is unique and is immutable for a given user.
```

Comment:
We want to list down what are the operations are supported, which is not in the list is by default not supported. We do't need to discuss about what are the attributes we don't support for patch.